src/include/mpi.h.in

```diff
@@ -568,6 +576,7 @@ extern int * const MPI_WEIGHTS_EMPTY MPICH_API_PUBLIC;
 #define MPI_TYPECLASS_REAL 1
 #define MPI_TYPECLASS_INTEGER 2
 #define MPI_TYPECLASS_COMPLEX 3
+#define MPI_TYPECLASS_LOGICAL 4
```

Comment:
Unfortunately, I think this one didn't make it for 5.0, but it may eventually land in 5.1. Should we keep it as it is or rename as `MPI_TYPECLASS_LOGICAL` ?

Reply:
BTW, `src/binding/abi/mpi_abi.h` currently lists `MPIX_TYPECLASS_LOGICAL`.

Reply:
I'll remove it for now (to do). It is trivial enough that we can add it when Forum approves it.